bridges/otellogr/logsink.go

```diff
@@ -201,9 +201,8 @@ var _ logr.LogSink = (*LogSink)(nil)
 // For example, commandline flags might be used to set the logging
 // verbosity and disable some info logs.
 func (l *LogSink) Enabled(level int) bool {
-	ctx := context.Background()
 	param := log.EnabledParameters{Severity: l.levelSeverity(level)}
-	return l.logger.Enabled(ctx, param)
+	return l.logger.Enabled(context.Background(), param)
```

Comment:
While not mandatory, the variable made sense too here, especially since it's rather a rare behavior to create empty contextes in here.